pulled from upstream absl, not yet in chromium -



commit 94d77fe3604dd86b5198f942110c240f01242439
Author: Derek Mauro <dmauro@google.com>
Date:   Tue Jun 20 12:58:51 2023 -0700

    Add missing #include <limits>
    
    Fixes #1482
    
    PiperOrigin-RevId: 542023050
    Change-Id: Iba712083edc9a24732a71f51be22ea970115809c

--- a/third_party/abseil-cpp/absl/debugging/internal/stacktrace_aarch64-inl.inc
+++ b/third_party/abseil-cpp/absl/debugging/internal/stacktrace_aarch64-inl.inc
@@ -13,6 +13,7 @@
 #include <cassert>
 #include <cstdint>
 #include <iostream>
+#include <limits>
 
 #include "absl/base/attributes.h"
 #include "absl/debugging/internal/address_is_readable.h"
